packages/ai-jsx/src/lib/anthropic.tsx

```diff
@@ -105,33 +105,47 @@ export async function* AnthropicChatModel(
   });
   yield AI.AppendOnlyStream;
   const messages = await Promise.all(
-    messageElements.filter(AI.isElement).map(async (message) => {
-      switch (message.tag) {
-        case UserMessage:
-          return `${AnthropicSDK.HUMAN_PROMPT}: ${await render(message)}`;
-        case AssistantMessage:
-          return `${AnthropicSDK.AI_PROMPT}: ${await render(message)}`;
-        case SystemMessage:
-          throw new AIJSXError(
-            'Anthropic models do not support SystemMessage. Change your user message to instruct the model what to do.',
-            ErrorCode.AnthropicDoesNotSupportSystemMessage,
-            'user'
-          );
-        case FunctionCall:
-        case FunctionResponse:
-          throw new AIJSXError(
-            'Anthropic models do not support functions.',
-            ErrorCode.AnthropicDoesNotSupportFunctions,
-            'user'
-          );
-        default:
-          throw new AIJSXError(
-            `ChatCompletion's prompts must be UserMessage or AssistantMessage, but this child was ${message.tag.name}`,
-            ErrorCode.ChatCompletionUnexpectedChild,
-            'internal'
-          );
-      }
-    })
+    messageElements
+      .filter(AI.isElement)
+      .flatMap((message) => {
+        if (message.tag === SystemMessage) {
+          return [
+            <UserMessage>For subsequent replies you will adhere to the following instructions: {message}</UserMessage>,
+            <AssistantMessage>Okay, I will do that.</AssistantMessage>,
```

Comment:
heh, does this help? I've experimented with stuff like this in the past but haven't ever rigorously tested it.

Reply:
It seemed to work well enough for the demos in `examples` that I tried 😂

Reply:
But like, you noticed a perf improvement with it?

Reply:
As compared to blowing up, yes :P

Langchain IIRC does something like `User: <admin>$SYSTEM_MESSAGE</admin>`, and this seemed to work better than that in the toy examples I tried 🤷

Reply:
I didn't mean compared to blowing up – I meant specifically just including the `AssistantMessage`.